packages/stryker/src/test-runner/ChildProcessTestRunnerWorker.ts

```diff
@@ -0,0 +1,46 @@
+import { TestRunner, TestRunnerFactory, RunnerOptions, RunOptions } from 'stryker-api/test_runner';
+import { errorToString } from '../utils/objectUtils';
+
+/**
+ * 
+ */
+export default class ChildProcessTestRunnerWorker implements TestRunner {
+
+  private underlyingTestRunner: TestRunner;
+
+  constructor(realTestRunnerName: string, options: RunnerOptions) {
+    this.underlyingTestRunner = TestRunnerFactory.instance().create(realTestRunnerName, options);
+  }
+
+  init(): Promise<void> | void {
+    if (this.underlyingTestRunner.init) {
+      return this.underlyingTestRunner.init();
+    } else {
+      return;
+    }
+  }
+
+  dispose() {
+    if (this.underlyingTestRunner.dispose) {
+      return this.underlyingTestRunner.dispose();
+    } else {
+      return;
+    }
+  }
+
+
+  async run(options: RunOptions) {
+    const result = await this.underlyingTestRunner.run(options);
+    // If the test runner didn't report on coverage, let's try to do it ourselves.
+    if (!result.coverage) {
+      result.coverage = (Function('return this'))().__coverage__;
```

Comment:
How does this work?

Reply:
It's the same as used by istanbul. It's a way to always grab the global object. I could change it to `global.__coverage__`. that should be fine. Agreed?

Reply:
Sounds good!